tests/on_wlm/test_wlm_orc_config_settings.py

```diff
@@ -0,0 +1,213 @@
+import pytest
+
+from smartsim import Experiment
+from smartsim.error import SmartSimError
+
+# retrieved from pytest fixtures
+if pytest.test_launcher not in pytest.wlm_options:
+    pytestmark = pytest.mark.skip(reason="Not testing WLM integrations")
+
+try:
+    from smartredis import Client
+
+    config_setter = Client.config_set
+except AttributeError:
+    pytestmark = pytest.mark.skip(reason="SmartRedis version is < 0.3.0")
+
+
+def test_config_methods_on_wlm(fileutils, wlmutils):
+    """test setting single node orchestrator configurations"""
+    launcher = wlmutils.get_test_launcher()
+
+    exp_name = "test_config_methods_on_wlm"
+    exp = Experiment(exp_name, launcher=launcher)
+    test_dir = fileutils.make_test_dir(exp_name)
+
+    db = wlmutils.get_orchestrator()
+    db.set_path(test_dir)
+
+    exp.start(db)
+    try:
+        db.enable_checkpoints(1)
+        db.set_max_memory("3gb")
+        db.set_eviction_strategy("allkeys-lru")
+        db.set_max_clients(50_000)
+        db.set_max_message_size(2_147_483_648)
```

Comment:
@MattToast thanks for the comments! In respect to the fixture and parameterize comments, I'll just go ahead and address all of them in this single comment. 

I've implemented `db` and `db_cluster` fixtures for automatic db startup and teardown. This helped reduce LOC by quite a bit, and might be something worth using in other parts of the test suite. I also added a `dbutils` fixture for access to convenience methods.

I also momentarily parameterized the tests, but after further inspection, this ended up adding around 4 minutes to the test suite run time since launching the orchestrator takes so long and parameterizing increases the number of db startups and teardowns. In my eyes, the added time doesn't justify the parameterizing in this situation, but let me know what you think; maybe in the future each parameter could be ran on a separate process?

I also removed the non-clustered tests from `test_wlm_orc_config_settings.py` since those exact tests will already run in `test_orc_config_settings.py`.

Reply:
@EricGustin Looks fantastic!

Regarding parameterization, I thought there was a way to set a fixture scope to all of the parameters in a parameterized function, but after reviewing the pytest docs, it appears I was mistaken. If we really wanted to we could set the fixture scope to a class level and wrap each of the parametrized tests in their own class, but I think that seems somewhat convoluted. Totally okay forgoing parameterization!!

All and all LGTM!!